### PR TITLE
Bug fixes

### LINKED ISSUE #5610: Success message pops up even when kubeconfig is not uploaded
#### Current Behavior
Even if there's no kubeconfig uploaded, there is a success message that pops up stating `Kube config successfully uploaded`. 

#### Expected Behavior
The uploaded button should ideally be disabled when there's nothing uploaded. 

- Server logs show `Failed to get Kubernetes context`

#### Screenshots/Logs

https://user-images.githubusercontent.com/53532328/173416218-cdff3b6a-795c-4931-bc65-be95df2dcc0e.mov

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**

This PR fixes #5610 #5613

**Notes for Reviewers**

https://user-images.githubusercontent.com/79860504/173499270-2cdcf1c1-ae01-47f3-8535-5d921948c712.mov

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by theBeginner86 — COMMENTED** [review]
Few things I observed while testing with 2 clusters: `docker-desktop` and `minikube`

- Searching doesn't work on deploy modal

https://user-images.githubusercontent.com/73700530/173449827-5381ff9b-3a1c-4217-a306-3c4aa44ac09e.mov



- If I don't select any cluster then it still deploys the service mesh in some random cluster.

https://user-images.githubusercontent.com/73700530/173449853-dab895b6-e3a1-44cd-b9c1-a4cbf9e95ac5.mov

**Comment by MUzairS15** [comment]
@theBeginner86 Searching issue in this PR??

**Comment by theBeginner86** [comment]
> @theBeginner86 Searching issue in this PR?? 

@MUzairS15 
Yeah this PR.

**Comment by MUzairS15** [comment]
Did you build-ui?? You are on 9081.

**Comment by theBeginner86** [comment]
> @theBeginner86 Did you build ui?? You are on 9081.

@MUzairS15 
Yeah I did build the UI but it wasn't working then. 
Could be due to caching issue as now searching does work after rebuilding the UI again 😅

**Comment by MUzairS15** [comment]
> LGTM! I think the ConfirmationModal is not working up until yet in Meshery-Extension due to this PR not landed yet! If the rest of comments are resolved, we are fine to merge.

Yes @Abhishek-kumar09 you are right.
@theBeginner86 any more concerns/bugs you encountered??

**Review by theBeginner86 — APPROVED** [review]
LGTM

File: ui/components/ConfirmationModal.js [hunk 1]
```diff
@@ -219,7 +267,13 @@ const mapStateToProps = state => {
...
 });
 
-export default withStyles(styles)(connect(mapStateToProps, mapDispatchToProps)(withSnackbar(ConfirmationMsg)));
\ No newline at end of file
+export default withStyles(styles)(connect(mapStateToProps, mapDispatchToProps)(withSnackbar(ConfirmationMsg)));
+
+
+// :
+//
+// }
```
Comment (Abhishek-kumar09) [inline ui/components/ConfirmationModal.js:279]: remove t
```suggestion
```

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by MUzairS15** [comment]
@Abhishek-kumar09

**Review by Abhishek-kumar09 — APPROVED** [review]
LGTM

**Pull request merged**